clisops/etc/roocs.ini

```diff
@@ -3,3 +3,4 @@ chunk_memory_limit = 250MiB
 
 [clisops:write]
 file_size_limit = 1GB
+output_staging_dir = /gws/smf/j04/cp4cds1/c3s_34e/rook_prod_cache
```

Comment:
@ellesmith88: I think this should probably be provided as an unset/empty option, e.g.:

`output_staging_dir =`

The `/gws/smf/...` directory is only relevant to the CEDA rook production deployment so it will be set in the `rook` ansible playbook and pushed on to the production nodes.  It is useful to keep the unset version here just so we have recorded it...and that relate to the issue we discussed about having a complete set of _possible_ `roocs.ini` contents listed somewhere - probably in `roocs-utils` in a markdown file explaining the configuration options (which would become part of the documentation under readthedocs).